packages/components/package.json

```diff
@@ -53,6 +53,7 @@
 		"@storybook/react": "^6.4.18",
 		"@testing-library/jest-dom": "^5.16.2",
 		"@testing-library/react": "^12.1.3",
+		"@testing-library/react-hooks": "^8.0.1",
```

Comment:
The rest of the monorepo is using `7.0.2` (tested with `npm ls @testing-library/react-hooks`) so I'll update the PR to use the older version. [There isn't a significant difference between the versions](https://github.com/testing-library/react-hooks-testing-library/releases).